arch/riscv/build.mk

```diff
@@ -17,27 +17,49 @@ DEFINES := -DF_CPU=$(F_CLK) \
            -DF_TIMER=$(F_TICK) \
            -include config.h
 
-ASFLAGS = -march=rv32imzicsr -mabi=ilp32
-CFLAGS += -Wall -Wextra -Wshadow -Wno-unused-parameter -Werror
+CROSS_COMPILE ?= riscv32-unknown-elf-
```

Comment:
Set the default value to `riscv-none-elf-` as [xPack GNU RISC-V Embedded GCC](https://xpack-dev-tools.github.io/riscv-none-elf-gcc-xpack/) is suggested.